assets/app/view/game/map.rb

```diff
@@ -47,7 +48,12 @@ def render
           left = (@tile_selector.x + map_x) * SCALE
 
           selector =
-            if @tile_selector.hex.tile != @tile_selector.tile
+            if @tile_selector.is_a? Lib::TokenSelector
```

Comment:
parens

Reply:
can we add a cop for this?

Reply:
i'm not sure, i spent a little trying to find a cop to no success.